airflow/bin/cli.py

```diff
@@ -410,18 +410,28 @@ def webserver(args):
         app.run(debug=True, port=args.port, host=args.hostname)
     else:
         pid, stdout, stderr, log_file = setup_locations("webserver", pid=args.pid)
+
+        ssl_certfile=configuration.get('webserver', 'ssl_certfile')
+        ssl_keyfile=configuration.get('webserver', 'ssl_keyfile')
+        secure_params = ssl_certfile and ssl_keyfile
+        if secure_params:
```

Comment:
Might want to break this up so that you can specify JUST ssl_certfile or JUST ssl_keyfile from args, and inherit from the config otherwise. Then validate that both ssl_certfile and ssl_keyfile are set.